apps/cow-fi/lib/hooks/useConnect.ts

```
@@ -2,21 +2,28 @@ import { useAccount, useConnect as useConnectWagmi } from 'wagmi'
 import { useCallback, useEffect, useState } from 'react'
 import { useConnectModal } from '@rainbow-me/rainbowkit'
 import { ConnectResult, PublicClient } from '@wagmi/core'
-import { clickOnMevBlocker } from '../../modules/analytics'
+import { initGtm } from '@cowprotocol/analytics'
```

ESLint

[error] There should be at least one empty line between import groups

(import/order)